databrowser/src/layouts/AppLayoutDefault.vue

```diff
@@ -12,7 +12,7 @@
 
 <script lang="ts">
 import { defineComponent } from '@vue/runtime-core';
-import AppHeader from '../components/header/AppHeader.vue';
+import AppHeader from './components/AppNavigation.vue';
```

Comment:
The import is named `AppHeader` but should probably be named `AppNavigation` to prevent confusion. Probably a leftover of refactoring.